share/wake/lib/system/job.wake

```diff
@@ -391,6 +391,13 @@ export def getJobStatus job = match (getJobReport job)
     else Signaled (-status)
 
 # Implement FUSE-based Runner
+# The FUSE runner on linux supports a few isolation options via resources:
+# - "isolate/user": the Job will appear to run as root
+# - "isolate/host": make the hostname appear to be "build.local"
+# - "isolate/net": disables network access
+# - "isolate/workspace": makes the build appear run in /var/cache/wake
+#    ... if /var/cache/wake does not exist, a directory 'build/wake' is
+#        used relative to where wake has been installed
```

Comment:
I'd request that this be a bit more precisely worded (and also can we add this all to a README?). I think we should make it clear _what_ appears to be in that dir and when. Here's a suggested rewording to capture what I think is missing:

> Make jobs appear to run in /var/cache/wake. Both the perceived workspace dir and the current working directory will be set to that directory. If /var/cache/wake does not exist, ... <remainder of your sentence>

Reply:
What README? This does not belong in the README.md of the wake project.
Do you mean in a file in the directory, saying why the directory exists?

Reply:
No, I think this needs to be in the README. Aren't the `isolate/*` resource strings meant to be used by Plan authors?

Reply:
What README are you talking about?
Do you mean the tutorial?

Reply:
I am using the term README a bit too loosely and incorrectly. What I mean is something documenting the Plan/Job/resource system.

I guess we don't have on that already.

Reply:
I see. I agree we should have documentation on the Plan/Job/resource system. It's on my TODO list.
However, this change sounds like a follow-up PR: write documentation.